python/utils/satcfg.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2019 Daniel Estevez <daniel@destevez.net>
+#
+# This file is part of gr-satellites
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+from pathlib import Path
+
+
+def get_cfg(norad):
+    """
+    Opens and returns the local configuration for selected satellite
+    """
+    config_dir = Path.home() / '.gr_satellites'
+    config_filename = config_dir / 'sat.cfg'
+
+    if not config_dir.exists():
+        Path.mkdir(config_dir)
```

Comment:
I think it's not necessary to create the `config_dir` if it doesn't exist, since in this case we will not use it at all. The `open_config()` function creates the `config_dir` because if `config.ini` doesn't exist, it creates it using a template, so the `config_dir` must exist to do this.